swift/swift.py

```diff
@@ -151,6 +140,7 @@ def destroyBus(self, name: str):
             name: A name of the bus to destroy.
         """
         self._buses[name].deleteLater()
+        self._buses.pop(name).stop()

```

Comment:
Hmm, maybe changing the order could be safer:
```python
self._buses[name].stop()
self._buses.pop(name).deleteLater()
```

Reply:
I made some modification to implement `destroyBus()` and `destroyApp()` similarly.